### PR TITLE
Supplemented and modified the new explanatory document to enhance readability

### PR DESCRIPTION
**Pull request by Appointat**
# Description

Supplemented and modified the new explanatory document to enhance readability
- Update README.md (Add more information about our project)
- Create SUPPORT.md
- Create TRANSLATIONS.md
- Create ISSUE_TEMPLATE.md

## Type of change

- [x] This change requires a documentation update

# How has this been tested?

The changes in these files are based on the reference below:
1. Referred [awesome-machine-learning](https://github.com/josephmisiti/awesome-machine-learning) for content table in README.md
2. Referred [OpenML](https://github.com/openml/OpenML) for the introduction including target users, goals, benefits and features
3. Referred [facebook/jest](https://github.com/facebook/jest) for the main framework of README.md
4. Refferd [mlcourse](https://github.com/davidrosenberg/mlcourse) for the introduction of the project
5. Refferd [ML-For-Beginners](https://github.com/microsoft/ML-For-Beginners) for the template of TRANSLATIONS.md
6. Refferd [Open Source Guide](https://opensource.guide) for the SUPPORT.md and ISSUE_TEMPLATE.md 

# Checklist:

- [x]  My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [x]  I have performed a self-review of my own code
- [x]  I have commented my code, particularly in hard-to-understand areas
- [x]  I have made corresponding changes to the documentation
- [x]  My changes generate no new warnings
- [x]  I have applied tests that prove my fix is effective or that my feature works
- [x]  New and existing unit tests pass locally with my changes
- [x]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ISSUE_TEMPLATE.md
Comment (zhangqi444) [inline ISSUE_TEMPLATE.md:1]: I think the title should be `# Description`, Instead of `# ISSUE_TEMPLATE`. You can test this once the code is merged.

File: SUPPORT.md
Comment (zhangqi444) [inline SUPPORT.md:1]: `**` is not necessary for title.

File: SUPPORT.md
Comment (zhangqi444) [inline SUPPORT.md:21]: Change the url placeholder to `https://TODO`

File: SUPPORT.md
Comment (zhangqi444) [inline SUPPORT.md:7]: Fix the url?

File: TRANSLATIONS.md
Comment (zhangqi444) [inline TRANSLATIONS.md:11]: The translated files should follow this naming convention, e.g.

File: TRANSLATIONS.md
Comment (zhangqi444) [inline TRANSLATIONS.md:26]: Thank you

**Review by zhangqi444 — COMMENTED** [review]
You should not rename the slides/introduction.md to README.md. The former will be compiled as part of the book. Instead, you should create a new README file if needed.

**Comment by zhangqi444** [comment]
Please help add the repo links you have referred to the PR description.

File: README.md
Comment (zhangqi444) [inline README.md:1]: Ocademy: an open academy for Data Science, Machine Learning, and Deep Learning

File: README.md
Comment (zhangqi444) [inline README.md:11]: Let's have only the first letter as upper case.

File: README.md
Comment (zhangqi444) [inline README.md:123]: capitalize the first letter of each word of data science, machine learning, and deep learning

File: README.md
Comment (zhangqi444) [inline README.md:117]: Add TODO as content of this section if you don't have anything to add for now.

File: README.md
Comment (zhangqi444) [inline README.md:119]: same here.

File: README.md
Comment (zhangqi444) [inline README.md:111]: Use the schedule link as this, https://github.com/open-academy/schedule

File: README.md
Comment (zhangqi444) [inline README.md:109]: remove such content,

```
Due to limited capacity and venue availability, the course will be conducted online via [online textbook](https://open-academy.github.io/machine-learning/intro.html) and [VooV (Tencent) meeting](https://www.tencentcloud.com/products/tvm), and at this time, we are only accepting students who are proficient in Chinese (although we plan to address this issue in the future).
```

File: README.md
Comment (zhangqi444) [inline README.md:54]: Please not hard code such content. Instead, just put the home page link of the open source book here.

**Review by zhangqi444 — COMMENTED** [review]
Great job. This is a good start.

File: ISSUE_TEMPLATE.md
Comment (Appointat) [reply ISSUE_TEMPLATE.md]: Thanks for the comment, I will modify it.

File: SUPPORT.md
Comment (Appointat) [reply SUPPORT.md:1]: Thanks for the comment, I will modify it.

File: SUPPORT.md
Comment (Appointat) [reply SUPPORT.md]: Thanks for the comment, I will modify it.

File: SUPPORT.md
Comment (Appointat) [reply SUPPORT.md]: Thanks for the comment, I will modify it.

File: TRANSLATIONS.md
Comment (Appointat) [reply TRANSLATIONS.md]: Thanks for the comment, I will modify it.

File: TRANSLATIONS.md
Comment (Appointat) [reply TRANSLATIONS.md:26]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md:116]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md:110]: Thanks for the comment and this link, but I don't know all the details of the course timetable. So I would like to leave it TO BE DONE and not modify the table as for now.

File: README.md
Comment (Appointat) [reply README.md]: Thanks for the comment, I will modify it.

File: README.md
Comment (Appointat) [reply README.md:53]: Thanks for the comment, I will modify it.

**Comment by Appointat** [comment]
Hello @zhangqi444 @reveurmichael, I have finished the first modification of the PR according to your nice and useful comments. Please review it again. Thanks a lot.

File: README.md
Comment (zhangqi444) [inline README.md:14]: Ocademy

**Review by zhangqi444 — CHANGES_REQUESTED** [review]

**Comment by zhangqi444** [comment]
Please make sure all the comments are resolved.

File: README.md
Comment (zhangqi444) [reply README.md:53]: Is this updated?

File: README.md
Comment (zhangqi444) [reply README.md:110]: Remove the table and just put the link here.

**Review by zhangqi444 — APPROVED** [review]

**Comment by Appointat** [comment]
> You should not rename the slides/introduction.md to README.md. The former will be compiled as part of the book. Instead, you should create a new README file if needed.

Yes, thanks, I have corrected the file name to `introduction.md`.

**Pull request merged**
